.github/workflows/lambda_release.yml

```
@@ -0,0 +1,35 @@
+name: Release
```

I don't think we can have two workflows named "Release". Maybe "Create Release Tags" is better?
```suggestion
name: Lambda Release
```

---

a billion percent. great catch!